mrbgem.rake

```diff
@@ -4,5 +4,43 @@ MRuby::Gem::Specification.new('mruby-redis') do |spec|
   spec.version = '0.0.1'
   spec.linker.libraries << "hiredis"
```

Comment:
line 5: a compiler found the system libraries of hiredis before linking `"#{hiredis_dir}/libhiredis.a"`, I guess. Could you try it after removing `line 5`?